Frameworks/UIKit/UIView.mm

```diff
@@ -409,6 +408,13 @@ - (UITouchPhase)_processPointerEvent:(WUXIPointerRoutedEventArgs*)pointerEventAr
             // Add this touch to the set of all current touches in the app
             [s_allTouches addObject:touchPoint.touch];
 
+            if (DEBUG_HIT_TESTING_LIGHT) {
+                TraceVerbose(TAG, L"Hit testing view %hs(0x%p).", object_getClassName(self), self);
+            }
+
+            // Update the hit-tested view. (Only one owner per UITouch lifetime)
+            touchPoint.touch->_view = [self _doHitTest:touchPoint.touch allTouches:s_allTouches];
```

Comment:
we shouldn't have to do another hit test here, as Xaml already chose the hit-test view..  what scenario is this covering?